package.json

```diff
@@ -91,6 +91,15 @@
     "@types/react": "^17.0.83",
     "@types/react-dom": "^17.0.25",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "webdriverio": {
+      "puppeteer-core": "^23.10.3"
+    },
+    "enzyme": {
+      "cheerio": "1.0.0-rc.10"
+    },
+    "chai-enzyme": {
+      "cheerio": "1.0.0-rc.10"
+    }
```

Comment:
Another fun override in this fix: enzyme lists 1.0.0-rc version with a caret, which makes npm dedupe it with stable version 1.0.0 that webdriverio uses, but this version actually doesn't work with enzyme, so we're overriding enzyme to use exact rc. Hopefully that's the last one that was causing issues 🤞 

I'm itching to just go through the handful of files that are still using enzyme and refactor those tests now...